### PR TITLE
added recipe flufl.enum

### PR DESCRIPTION
**Pull request by cehbrecht**
this recipe is need by pywps 4.x (PR #2295) for python 2.7 builds.

### CODE REVIEW

File: recipes/flufl.enum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package:
+  name: flufl.enum
+  version: "4.1.1"
```
Comment (ocefpaf) [inline recipes/flufl.enum/meta.yaml]: Take a look at the example and switch the 4.1.1 in various places to the jinja variable to improve the recipe maintainability later on.

File: recipes/flufl.enum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+source:
+  fn: flufl.enum-4.1.tar.gz
+  url: https://pypi.python.org/packages/65/51/aa79e41a6deefab555a7cc26a917363c67c6b7e2e1b43067f3d11b5e1b7e/flufl.enum-4.1.1.tar.gz
```
Comment (ocefpaf) [inline recipes/flufl.enum/meta.yaml]: Please change this to the more readable `https://pypi.io/packages/source/f/flufl.enum/flufl.enum-{{ version }}.tar.gz` URL.

**Comment by ocefpaf** [comment]
Closing and re-opening to see if you can trigger Travis-CI.

**Comment by cehbrecht** [comment]
@ocefpaf 
maybe the osx queue is not available on travis? I could disable the osx build for now to get a step further.

**Comment by ocefpaf** [comment]
> maybe the osx queue is not available on travis?

It seems that Travis-CI fixed the outage from the past feels days.

> I could disable the osx build for now to get a step further.

No need because we still rely on Travis-CI to create the feedstock. You can remove that only if you are unsure the package will build properly on OS X. (But it looks like it will so the best think to do is to wait :unamused:)

**Pull request merged**